### DISCUSSION TITLE
the following blocks (ids) do not exist on the mesh

### DISCUSSION THREAD

**Discussion by selarem** [idaholab/moose#23458] [Q&A Modules: Solid mechanics] [answered]
Hello Moosers,
I am trying to define different elastic properties for different elsets (layers); the layers are elsets defined in my mesh file, however i got the error message :+1: 
```
the following blocks (ids) do not exist on the mesh 
```
A block is not an Elset ?

regards,

Sab

**Comment by hugary1995** [comment]
~Use boundary = elset~

**Comment by selarem** [comment]
I am using this syntax:
```
[./elasticity_tensor14]
    type = ComputeElasticityTensorCP
   block='c1'   
    C_ijkl = '11e3 12e3 13e3 22e3 23e3 33e3 44e3 55e3 66e3'
    fill_method = symmetric9
    euler_angle_1 = 0.0
    euler_angle_2 = .0
    euler_angle_3 = 0.0
  [../]
```
where to add  boundary = elset ? 
thanks

**Reply by hugary1995** [reply]
I may have misunderstood. What does elset mean?

**Reply by selarem** [reply]
Element set

**Reply by hugary1995** [reply]
That sounds like an Abaqus keyword. I think `block` should work. Someone else more familiar with the Abaqus mesh format may be able to help.

**Reply by GiudGiud** [reply]
Open your mesh with paraview and look at the blocks in the properties tab on the left. This is what you can use as a 'block'

I m not sure what an element set is wrt MOOSE. Maybe @dschwen

**Reply by hugary1995** [reply, marked as answer]
In case paraview doesn't support your mesh format but moose does, you can do `my-app-opt -i input.i --mesh-only` to generate an exodus file.

**Comment by selarem** [comment]
Thank, it works fine non.
Regards
